Dockerfile.dev

```
@@ -4,12 +4,12 @@ SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
 # Uninstall pre-installed formatting and linting tools
 # They would conflict with our pinned versions
-RUN pipx uninstall black
-RUN pipx uninstall flake8
-RUN pipx uninstall pydocstyle
-RUN pipx uninstall pycodestyle
-RUN pipx uninstall mypy
-RUN pipx uninstall pylint
+RUN \
+    pipx uninstall black && \
+    pipx uninstall flake8 && \
+    pipx uninstall pydocstyle && \
+    pipx uninstall mypy && \
+    pipx uninstall pylint
```

Our preferred formatting is to have the ampersands at the start of the line, not at the end.

```suggestion
RUN \
    pipx uninstall black \
    && pipx uninstall flake8 \
    && pipx uninstall pydocstyle \
    && pipx uninstall mypy \
    && pipx uninstall pylint
```

---

yeah, I don't get why people do that, it looks ugly and breaks alignment as you can see :) but sure, no problem :p